private_gpt/ui/ui.py

```diff
@@ -123,10 +129,10 @@ def yield_deltas(completion_gen: CompletionGen) -> Iterable[str]:
                 used_files = set()
                 for index, source in enumerate(cur_sources, start=1):
                     if f"{source.file}-{source.page}" not in used_files:
-                        sources_text = (
-                            sources_text
-                            + f"{index}. {source.file} (page {source.page}) \n\n"
-                        )
+                        if settings().USE_HYPERLINKS_FOR_SOURCES:
```

Comment:
there is a ui setting object. The best idea is create a new filed in UI settings and create a boolean property